.github/workflows/deploy_staging.yml

```diff
@@ -0,0 +1,15 @@
+on:
+  push:
+    branches: [main]
+
+jobs:
+  deploy:
+    name: Deploy to staging environment
+    uses: ./.github/workflows/deploy.yml
+    with:
+      deploy_env: stage
```

Comment:
think we want to use `staging` to be consistent with other backend stacks we have